SpatialGDK/Source/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -127,11 +127,13 @@ void USpatialWorkerConnection::ConnectToLocator()
 		if (DeploymentList->error != nullptr)
 		{
 			UE_LOG(LogSpatialWorkerConnection, Error, TEXT("Error fetching deployment list: %s"), UTF8_TO_TCHAR(DeploymentList->error));
```

Comment:
`"Error fetching deployment list: %s. Unable to establish connection to SpatialOS."`

Reply:
Done, with different phrasing.